### ISSUE TITLE
New project includes reference to the .templateconfig

### ISSUE THREAD

**Issue by dansiegel** [ASP-NET-Core-Boilerplate/Templates#180]
After creating a new project using the dotnet template the .templateconfig folder is referenced in the csproj

```xml
  <ItemGroup>
    <Content Include=".template.config\dotnetcli.host.json" />
    <Content Include=".template.config\template.json" />
  </ItemGroup>
```

﻿**Which project template is the issue about?**

dotnet new api template

**Which version of the project template is the issue about?**

1.6.3

**Which version of Visual Studio or the dotnet SDK were you using?**

2.0.1-servicing-006924

**Comment by RehanSaeed** [comment]
There were breaking changes in 2.0. I'm working on a 2.0 update for the framework and templates. Hope to have it out soon'ish.

**Comment by dansiegel** [comment]
Pretty sure that is unrelated. The issue is that the actual template's csproj includes a reference to the .template.config. You can see it [here](https://github.com/ASP-NET-Core-Boilerplate/Templates/blob/master/Source/Boilerplate.Templates/Content/ApiTemplate/ApiTemplate.csproj#L32-L35) on line's 32-35. The result is that when a project is created from the template the new project's csproj has a reference to files that do not exist...

**Comment by RehanSaeed** [comment]
You look to be correct, not sure how that got there. Will get it fixed, hopefully this weekend.

**Comment by RehanSaeed** [comment]
Released 1.7.0 much much later than planned but better late than never.
